com.unity.netcode.gameobjects/Runtime/Core/NetworkObject.cs

```diff
@@ -320,7 +332,7 @@ public void NetworkHide(ulong clientId)
                 var bytesReported = NetworkManager.LocalClientId == clientId
                         ? 0
                         : size;
-                NetworkManager.NetworkMetrics.TrackObjectDestroySent(clientId, NetworkObjectId, name, bytesReported);
+                NetworkManager.NetworkMetrics.TrackObjectDestroySent(clientId, NetworkObjectId, GetNameForMetrics(), bytesReported);
```

Comment:
Could `TrackObjectDestroySent` / `TrackOwnershipChangeReceived` just take the networkObject and then on the receiving end pull the `.networkObjectId` and `.GetNameForMetrics()`?

Reply:
Good call, that is a lot simpler